texts/depositing/overview.yml

```
@@ -7,4 +7,10 @@ description:
   # Displays when we have deposit compliance calculated
   compliance-level: '{{amount}}% of outputs are non-compliant.'
 
+  # Displays when compliance level is 100%
+  all-compliant: We couldn't detect any non-compliant outputs.
```

I am not sure about informal language (_couldn't_) but let's keep it.

```suggestion
  all-compliant: We couldn't detect any non-compliant output.
```